types/react-syntax-highlighter/index.d.ts

```diff
@@ -61,27 +65,33 @@ declare module 'react-syntax-highlighter/dist/esm/prism-async-light' {
     import { SyntaxHighlighterProps } from 'react-syntax-highlighter';
     export default class SyntaxHighlighter extends React.Component<SyntaxHighlighterProps> {
         static registerLanguage(name: string, func: any): void;
+        static supportedLanguages: string[];
     }
 }
 
 declare module 'react-syntax-highlighter/dist/esm/prism-async' {
     import * as React from 'react';
     import { SyntaxHighlighterProps } from 'react-syntax-highlighter';
-    export default class SyntaxHighlighter extends React.Component<SyntaxHighlighterProps> {}
+    export default class SyntaxHighlighter extends React.Component<SyntaxHighlighterProps> {
+        static supportedLanguages: string[];
+    }
 }
 
 declare module 'react-syntax-highlighter/dist/esm/prism-light' {
     import * as React from 'react';
     import { SyntaxHighlighterProps } from 'react-syntax-highlighter';
     export default class SyntaxHighlighter extends React.Component<SyntaxHighlighterProps> {
         static registerLanguage(name: string, func: any): void;
+        static supportedLanguages: string[];
```

Comment:
This property is not available on prism-light: https://github.com/react-syntax-highlighter/react-syntax-highlighter/blob/v13.5.3/src/prism-light.js